bullet_train-outgoing_webhooks/app/controllers/account/webhooks/outgoing/deliveries_controller.rb

```diff
@@ -63,17 +63,13 @@ def destroy
 
   # Never trust parameters from the scary internet, only allow the white list through.
   def delivery_params
-    strong_params = params.require(:webhooks_outgoing_delivery).permit(
+    params.require(:webhooks_outgoing_delivery).permit(
       :event_id,
       :endpoint_url,
       :delivered_at,
       # 🚅 super scaffolding will insert new fields above this line.
       # 🚅 super scaffolding will insert new arrays above this line.
     )
-
-    assign_date_and_time(strong_params, :delivered_at)
     # 🚅 super scaffolding will insert processing for new fields above this line.
```

Comment:
I think we've broken super scaffolding's ability to insert lines since the code expects a `strong_params` local variable: https://github.com/bullet-train-co/bullet_train-core/pull/334/files#diff-eda613639ab27832251c13ff26e97c46614576602167c5be468c0a5639160d3dR1111

@jagthedrummer @gazayas do either of you know if super scaffolding can/will insert lines in this controller?

Reply:
@kaspth I honestly don't know enough about how super scaffolding works under the hood to be able to make a confident assessment. But based on the line you linked to (and given my limited context) I'd say that you're likely right that removing the `strong_params` local variable will have some unintended consequences. Is there a good/easy way to write a test for that? (Both to confirm or disconfirm your hunch, and to make sure we don't break it in the future.)

Reply:
> Is there a good/easy way to write a test for that?

I'm not sure. @gazayas do you know how to write Super Scaffolding tests for something like the above?

Reply:
@kaspth @jagthedrummer
Thanks for pointing this out! I see what you're saying, so I have a couple of questions about how to handle this, and depending on that we can either add `strong_params = ...` back or keep the current changes.

## Description
We recently merged this PR: #330.

Since we've been slowly moving Bullet Train logic out of the starter repository and into packages, I feel like sometimes there is cruft that gets leftover that we simply moved over as-is because the code was working, which is why I was ok with deleting the TODO in the PR mentioned above. The strong parameters here for example were originally just in the API controllers in the starter repository if my memory serves me correctly. For all other scaffolded helpers like `assign_date` here, you're right, we would need `strong_params` for any other scaffolded methods to work. Since there are no helpers here anymore though, I don't think we need it. The `strong_params` handlers are pretty much the same across the board for different controllers, so I can understand if this was just copied over to get things working for the time being.

## My initial thoughts on the change
The current code is fine as-is because we can't Super Scaffold to `deliveries_controller.rb` since it's in `bullet_train-core`, and it does not serve as a template which is copied over like `tangible_things_controller.rb`, etc.

## Questions
However, technically speaking, a developer could eject this file to their own project if they wanted to.
1. Do we want to be able to Super Scaffold to this file after a developer were to eject it?
2. Also, we have `bin/super-scaffold incoming-webhooks` but nothing for outcoming webhooks. Should we add a scaffolder for outcoming webhooks? If so, I can write tests for it. I can't say I'm too familiar with that end of things to make an executive decision concerning it, but if we need it I'd be glad to look into and work on it.

Reply:
🤔 I'm gonna need to absorb this and think about it a bit.

Reply:
Same here, I saw that you shared a similar thought in #385 so I'll try to keep that in mind too.

Reply:
I didn't expect this to generate quite this much context, which is ok! But I'm thinking we could just leave this as is for now and revisit later. Like @gazayas mentioned the controller is unlikely to be super scaffoldable.

Reply:
@gazayas @kaspth, just so you know I ended up reverting this PR in https://github.com/bullet-train-co/bullet_train-core/pull/389. It was breaking tests in the starter repo and it seems like it had legitimately broken date and date/time pickers. I was able to generate a revert PR without a ton of trouble, so I figured a rollback was better than trying to sort out exactly what was wrong while everything remained in a broken state.

I generated this PR to revert the revert so that someone can pick back up with this: https://github.com/bullet-train-co/bullet_train-core/pull/390